src/components/QueryOutput/QueryOutput.js

```diff
@@ -558,7 +558,7 @@ export class QueryOutput extends React.Component {
 
   hasError = (response) => {
     try {
-      const referenceIdNumber = Number(response.data.reference_id.split('.')[2])
+      const referenceIdNumber = Number(response.data?.reference_id?.split('.')[2])
```

Comment:
This logic works, but it's neither standard nor ideal.

Have you tried either of these options? 

`if (response.status >= 200 && response.status < 300) {
  return false;
}`

Or

`if (response.data?.message === "Success") {
  return true;
}`